packages/lib/server-only/pdf/insert-field-in-pdf-v1.ts

```
@@ -41,9 +43,8 @@ export const insertFieldInPDFV1 = async (pdf: PDFDocument, field: FieldWithSigna
    * Blue box is the adjusted field width, height and position. It will represent
    * where the text will overflow into.
    */
-  const isDebugMode =
-    // eslint-disable-next-line turbo/no-undeclared-env-vars
-    process.env.DEBUG_PDF_INSERT === '1' || process.env.DEBUG_PDF_INSERT === 'true';
+  // biome-ignore lint/nursery/noUndeclaredEnvVars: DEBUG_PDF_INSERT is a local debugging flag.
```

Biome (2.5.0)

[error] Failed to parse category "lint/nursery/noUndeclaredEnvVars". Example of suppression: // biome-ignore lint: reason

(suppressions/parse)